components/delete_modal_trigger.test.tsx

```diff
@@ -5,16 +5,16 @@ import React from 'react';
 import {shallow} from 'enzyme';
 
 import Constants from 'utils/constants';
-import DeleteModalTrigger from 'components/delete_modal_trigger.jsx';
+import DeleteModalTrigger from 'components/delete_modal_trigger';
 import ConfirmModal from 'components/confirm_modal.jsx';
 
 describe('components/DeleteModalTrigger', () => {
     test('should throw error when trying to construct DeleteModalTrigger', () => {
         expect(() => {
-            new DeleteModalTrigger(); //eslint-disable-line no-new
+            new DeleteModalTrigger({onDelete: () => {}}); //eslint-disable-line no-new
```

Comment:
```suggestion
            new DeleteModalTrigger({onDelete: jest.fn()}); //eslint-disable-line no-new
```

Can we do this for consistency?

Reply:
Sorry, i had to reinstall windows cuz i had a virus, and i was trying to set up windows server setup, but it staking forever, just going to go with Windows WSL instead. I will get to it asap